pennylane/qnode.py

```diff
@@ -131,6 +131,36 @@ def _append_op(self, op):
                 raise QuantumFunctionError('State preparations and gates must precede measured observables.')
             self.queue.append(op)
 
+    def print_applied(self):
+        """Prints the most recently applied operations from the QNode."""
+        if not self.ops:
+            print("QNode has not yet been executed.")
+            return
+
+        print("Operations")
+        print("==========")
+        for op in self.queue:
+            if op.parameters:
+                params = ", ".join([str(p) for p in op.parameters])
+                print("{}({}, wires={})".format(op.name, params, op.wires))
+            else:
+                print("{}(wires={})".format(op.name, op.wires))
+
+        return_map = {
+            qml.operation.Expectation: "expval",
+            qml.operation.Variance: "var",
+            qml.operation.Sample: "sample"
+        }
+
+        print("\nObservables")
+        print("===========")
+        for op in self.ev:
```

Comment:
Just out of curiosity, are the names of `self.queue` and `self.ev` changed in the new QNode? I think they are very suboptimal.

Reply:
They should be changed, yes; feel free to double check in #354